probe-rs/src/rtt.rs

```diff
@@ -313,7 +313,9 @@ impl Rtt {
         for (i, b) in up_channels_buffer.into_iter().enumerate() {
             let buffer_size = b.size() as u64;
 
-            if let Some(chan) = Channel::from(core, i, ptr + offset, b)? {
+            if let Some(chan) = Channel::from(core, i, ptr + offset, b).map_err(|_| {
```

Comment:
The errors that are returned should not be thrown away but rather be added to the stack.